string RobotID
string AffiliatedModuleID
---
bool success